tests/integration_tests/test_integration_dds.py

```diff
@@ -209,6 +217,52 @@ def test_can_stage_and_deliver_force_flowcells(self):
                 status_response = yield self.http_client.fetch(link)
                 self.assertEqual(json.loads(status_response.body)["status"], StagingStatus.staging_successful.name)
 
+    @gen_test
+    def test_can_organise_stage_and_deliver_force_flowcells(self):
+        with tempfile.TemporaryDirectory(dir='./tests/resources/runfolders/',
+                                         prefix='160930_ST-E00216_0555_BH37CWALXX_') as tmpdir1, \
+                tempfile.TemporaryDirectory(dir='./tests/resources/runfolders/',
+                                            prefix='160930_ST-E00216_0556_BH37CWALXX_') as tmpdir2:
+            # First organise
+            unorganised_runfolder1 = self.create_unorganised_test_runfolders(tmpdir1)
+            unorganised_runfolder2 = self.create_unorganised_test_runfolders(tmpdir2)
+            
+            self._create_runfolder_structure_on_disk(unorganised_runfolder1)
+            self._create_runfolder_structure_on_disk(unorganised_runfolder2)
+
+            url = "/".join([self.API_BASE, "organise", "runfolder", unorganised_runfolder1.name])
+            response1 = yield self.http_client.fetch(self.get_url(url), method='POST', body='')
+            self.assertEqual(response1.code, 200)
+           
+            url = "/".join([self.API_BASE, "organise", "runfolder", unorganised_runfolder2.name])
+            response2 = yield self.http_client.fetch(self.get_url(url), method='POST', body='')
+            self.assertEqual(response2.code, 200)
+ 
+            # Then stage it
+            url = "/".join([self.API_BASE, "stage", "project", 'runfolders', 'JKL_123'])
+            payload = {'delivery_mode': 'FORCE'}
+            response_forced = yield self.http_client.fetch(self.get_url(url), method='POST', body=json.dumps(payload))
+            self.assertEqual(response_forced.code, 202)
+
+            response_json = json.loads(response_forced.body)
+
+            staging_status_links = response_json.get("staging_order_links")
+            staging_order_ids = response_json.get("staging_order_ids")
+
+            # Assert the staged folder structure has only one runfolder folder
+            temp_staging_dir = f"/tmp/{staging_order_ids.get('JKL_123')}/JKL_123"
+            for runfolder in os.listdir(temp_staging_dir):
+                self.assertFalse(set(runfolder).issuperset(set(os.listdir(f"{temp_staging_dir}/{runfolder}"))))
+            
+            # Insert a pause to allow staging to complete
+            time.sleep(1)
```

Comment:
I guess it would make sense to have this line + the successful check (line 258-264) before verifying that the staged runfolders don't contain a runfolder directory.